tests/command_line/test_refine_bravais_settings.py

```diff
@@ -11,27 +12,29 @@
 from dials.command_line import refine_bravais_settings
 
 
-def test_refine_bravais_settings_i04_weak_data(dials_regression, tmpdir):
+def test_refine_bravais_settings_i04_weak_data(dials_regression, tmp_path):
     data_dir = os.path.join(dials_regression, "indexing_test_data", "i04_weak_data")
     pickle_path = os.path.join(data_dir, "indexed.pickle")
     experiments_path = os.path.join(data_dir, "experiments.json")
-    with tmpdir.as_cwd():
-        refine_bravais_settings.run(
-            [
-                pickle_path,
-                experiments_path,
-                "reflections_per_degree=5",
-                "minimum_sample_size=500",
-                "beam.fix=all",
-                "detector.fix=all",
-                "prefix=tst_",
-            ]
-        )
+    result = procrunner.run(
```

Comment:
This `codecov` warning is ... odd?

Reply:
This test uses dials_regression hence isn't included in the codecov statistics.